lib/msg.js

```diff
@@ -1,10 +1,53 @@
-const { send } = require('./slack');
+const { sendMsg, sendBlocks } = require('./slack');
 
 const sendDigest = async (newEmojis) => {
-  const emojis = newEmojis.map((emoji) => `:${emoji}: — \`:${emoji}:\``).join('\n');
-  msg = `There are *${newEmojis.length}* new emojis today! :meow_happy_paws: \n${emojis}`;
+  const emojis = Object.keys(newEmojis).map((emoji) => {
+    if (newEmojis[emoji].includes('alias:')) {
+      return {
+        "type": "section",
+        "text": {
+          "type": "mrkdwn",
+          "text": `:${emoji}: — \`:${emoji}:\` (alias for \`:${newEmojis[emoji].split(':')[1]}:\`)`
+        }
+      }
+    }
 
-  return await send(msg);
+    return {
+      "type": "section",
+      "text": {
+        "type": "mrkdwn",
+        "text": `:${emoji}: — \`:${emoji}:\``
+      },
+      "accessory": {
+        "type": "image",
+        "image_url": newEmojis[emoji],
+        "alt_text": `emoji image for :${emoji}:`
+      }
+    }
+  }).sort((a, b) => {
+    if (!a['accessory'] && b['accessory']) {
```

Comment:
This sorting is so we move all the "alias" blocks to the bottom. `accessory` is the object that holds the image.